resources/views/emails/training_ticket.blade.php

```diff
@@ -10,6 +10,8 @@
     <li><b>Position:</b> {{ $ticket->position_name }}</li>
     <li><b>Date and Time:</b> {{ $ticket->date }} from {{ $ticket->start_time }} to {{ $ticket->end_time }} ET</li>
 	<li><b>Duration:</b> {{ $ticket->duration }}</li>
+	<li><b>Score (1-5):</b> @if($ticket->score) {{ $ticket->score }} @else N/A @endif</li>
```

Comment:
For UX, we need to provide some context behind the score (ex. 5 is best, 1 is worst)